src/openqaoa-azure/pyproject.toml

```diff
@@ -25,6 +25,7 @@ dependencies = [
     "qiskit-qir",
     "qiskit-ionq",
     "azure-quantum[qiskit]",
+    "qdk"
```

Comment:
Is this needed? I thought `qdk` was included by default in `azure-quantum`

Reply:
You are right. I was trying to see if the problem in the tests was related to missing dependencies and found that qdk was missing, but I guess it's not related to that...